retrofit/src/main/java/retrofit/client/Request.java

```diff
@@ -56,8 +51,8 @@ public String getUrl() {
     return url;
   }
 
-  /** Returns an unmodifiable list of headers, never {@code null}. */
-  public List<Header> getHeaders() {
+  /** Returns an unmodifiable list of headers. May be {@code null}. */
```

Comment:
nits: its _always_ unmodifiable, and it's not a list

non-nit: return empty headers instead of null?